notebooks/lensed_quasar_population_with_variability.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # load your psf kernel and transform matrix. If you have your own psf, please provide
</pre><p>is there somewhere a PSF the user can use, i.e. from the data directory? This should run without a non-accessible PSF</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>We do not have PSF anywhere outside of slsim. The PSF used here is in the TestData. I changed the path so that user do not need pass their path here.</p>